.pre-commit-config.yaml

```diff
@@ -0,0 +1,63 @@
+---
+repos:
+  - repo: git://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.1.1
+    hooks:
+      - id: remove-tabs
+
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v2.0.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: name-tests-test
+      - id: check-added-large-files
+      - id: check-byte-order-marker
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-json
+      - id: check-symlinks
+      - id: detect-private-key
+      - id: check-ast
+      - id: debug-statements
+
+  - repo: git://github.com/pycqa/pydocstyle.git
+    rev: 5.1.1
+    hooks:
+      - id: pydocstyle
+        exclude: '^tests*'
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: check-toml
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.770
+    hooks:
+      - id: mypy
+        exclude: '^(docs|tasks|tests)|setup\.py'
+        args: [--ignore-missing-imports]
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+
+  # Enable this in repositories with python packages.
+  # - repo: https://github.com/mgedmin/check-manifest
+  #   rev: '0.39'
+  #   hooks:
+  #     - id: check-manifest
+
+  - repo: https://gitlab.com/PyCQA/flake8
+    rev: '3.7.8'
+    hooks:
+      - id: flake8
+        additional_dependencies: ['pep8-naming']
+        # Ignore all format-related checks as Black takes care of those.
+        args: ['--ignore', 'E2,W5', '--select', 'E,W,F,N', '--max-line-length=136']
```

Comment:
changed `'--max-line-length=136'` to allow more characters for the description: https://github.com/thoth-station/invectio/blob/master/setup.py#L54